src/archive/src/main.rs

```diff
@@ -575,6 +575,7 @@ fn http_request(req: HttpRequest) -> HttpResponse {
 }
 
 fn encode_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> std::io::Result<()> {
+    const WASM_PAGE_SIZE_IN_BYTES: f64 = 65536.0;
```

Comment:
There should be a constant for that in the `cdk`. Also, as a side-note: I would not do floating point calculations unless absolutely necessary. Floating point calculations could introduce rounding errors.

Reply:
Nice! Fixed!